person.rb

```diff
@@ -1,4 +1,6 @@
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
   attr_reader :id
   attr_accessor :name, age
```

Comment:
- [ ] Kindly ensure your syntax is correct. Your Person class threw up errors here. You should have `:age` here (with `:`)